environments/environment_factory.py

```diff
@@ -0,0 +1,59 @@
+"""Environment factory class. Given a valid environment name and its constructor args, returns an instantiation of it
+"""
+from .environments import Environment
+# TODO(Oleguer): Fix this, its not very nice to have to import all classes
+from .pendulum import Pendulum
+# from spring import Spring
```

Comment:
Should we import all the implemented environments here?

Reply:
yes, not ideal